src/app/dataValidator/index.js

```diff
@@ -9,8 +10,43 @@ const validateData = data => {
   log('\nValidation complete!');
 };
 
-const data = require('../../../data/scenario-01.json');
+const validateFile = filename => {
+  const data = fs.readFileSync(filename);
+  validateData(JSON.parse(data));
+};
+
+const readAllFiles = (filenames, dirname) =>
+  filenames.forEach(filename => {
+    // explicitly ignore scenario-23
+    if (filename.includes('scenario-23.json')) {
+      return;
+    }
+
+    if (filename.includes('.json')) {
+      validateFile(`${dirname}/${filename}`);
+    }
+  });
 
-validateData(data);
+const readdirAsync = dirname =>
+  new Promise((resolve, rejects) => {
+    fs.readdir(dirname, (err, filenames) => {
+      if (err) {
+        rejects(err);
+      } else {
+        resolve(readAllFiles(filenames, dirname));
```

Comment:
I think that here this has a single promise that will return for the group of files. 
I think it would be better to use `Promise.all()` so that this Promise resolves when all of the Promises in the iterable resolve (i.e. all files are validated).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
I saw that in your PR description that you want to synchronously validate the json files - why is that? It seems better to do it asynchronously.

Reply:
Synchronously was a first draft at getting it working but I'm going to try a Promise.all and async now 😄